data/sles4sap/script_output.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: all
+  remote_user: cloudadmin
```

Comment:
I'm wondering if the remote_user can be also an argument of the function to make this more general. I suppose it can be overwritten via the CLI. And if it's more general, we could even move it out of `data/sles4sap/` ... just thinking out loud, not requesting a change here.